_assets/stylesheets/pages/_our-history.scss

```diff
@@ -0,0 +1,4 @@
+.our-history-bg {
+    background-image: url("https://images.ctfassets.net/y3a9myzsdjan/4tq2Pb9tFyEXs6kEexpLK7/cab00ea4d82d21fe461836b3aa9792ba/crossroads-church-old2.jpg");
+    height: 455px;
```

Comment:
Do you need to set this height? This could be done by using classes like `soft-ends` inside of the jumbotron.

Reply:
`soft-ends` doesn't quite get the job done. I just set the height per the design which has a good aesthetic on desktop and mobile.